cmd/harborAPI.go

```diff
@@ -548,3 +549,41 @@ func CatalogCustoms(shipment string, env string, buildToken string, catalogReque
 		log.Fatal("customs/catalog failed")
 	}
 }
+
+// Restart will create an EnvVar on a shipment that will allow it be restarted
+func Restart(username string, token string, shipment string, env string) {
+	if Verbose {
+		log.Printf("restarting shipment")
+	}
+
+	//build URI
+	var config = GetConfig()
+	values := make(map[string]interface{})
+	values["shipment"] = shipment
+	values["env"] = env
+	template, _ := uritemplates.Parse(config.ShipitURI + "/v1/shipment/{shipment}/environment/{env}/envVar/RESTART")
+	uri, _ := template.Expand(values)
+	if Verbose {
+		log.Printf("deleting: " + uri)
+	}
+
+	res, _, _ := delete(username, token, uri)
```

Comment:
just following current logic. Once shipit bulk is in, we can make this much simpler. Also, you can't get an envVar currently. I could just write this to use the bulk api, but that probably should all be done at once, so things are clean.